bundles/org.openhab.binding.myuplink/README.md

```diff
@@ -0,0 +1,79 @@
+|---------------------|---------------------|-----------------------------------------------------------------------------------------------|
+| bridge              | account             | cloud connection to a myUplink user account                                                   |
+| thing               | genericDevice       | the physical heatpump which is connected to myUplink                                          |
+
+
+## Discovery
+
+Auto-discovery is supported and will discover all heatpumps within an account and also detect the specific channels supported by the model.
+
+## Bridge Configuration
+
+The following configuration parameters are available for the binding/bridge:
+
+| Configuration Parameter | Required | Description                                                                                                                                                                                 |
+|-------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| clientId                | yes      | The clientId to login at myUplink cloud service. This is some kind of UUID. Visit https://dev.myuplink.com/apps to generate login credentials.                                              |
+| clientSecret            | yes      | The secret which belongs to the clientId.                                                                                                                                                   |
+| dataPollingInterval     | no       | Interval (seconds) in which live data values are retrieved from the Easee Cloud API. (default = 60)                                                                                         |
+
+## Thing Configuration
+
+It is recommended to use auto discovery which does not require further configuration.
+If manual configuration is preferred you need to specify configuration as below.
+
+| Configuration Parameter | Required | Description                                                                                                            |
+|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------|
+| deviceId                | yes      | The id of the heatpump that will be represented by this thing. Can be retrieved via API call or autodiscovery.         |
+
+
+## Channels
+
+The binding only supports all channels which are explicitely exposed by the myUplink API.
+Depending on your model and additional hardware the channels might be different. 
+Thus no list is provided here.
+
+## Full Example
+
+_Provide a full usage example based on textual configuration files._
+_*.things, *.items examples are mandatory as textual configuration is well used by many users._
+_*.sitemap examples are optional._

```

Comment:
TODO

Reply:
documentation is updated. Basically the example was already there I just added a missing parameter and added a short explaination.